odoo/addons/base/models/ir_actions.py

```diff
@@ -508,20 +570,6 @@ class IrActionsServer(models.Model):
     _order = 'sequence,name,id'
     _allow_sudo_commands = False
 
-    DEFAULT_PYTHON_CODE = """# Available variables:
```

Comment:
Do we indicate the available variables somewhere? maybe a link to the doc in the view?